web/src/components/netflow-traffic.tsx

```diff
@@ -143,6 +155,10 @@ export const NetflowTraffic: React.FC<{
 
   usePoll(tick, interval);
 
+  React.useEffect(() => {
```

Comment:
just for confirmation: this is going to be fetched just once, upon page loading, right?

Reply:
Actually, seems like there's an issue, it seems fetching the config in an infinite loop (I'm testing and seeing this right now)

Reply:
Fixed, thanks!

Reply:
hmm it's weird, it seemed fixed at first, but now I'm seeing it again .. i'll investigate

Reply:
I've done some tests and it works pretty well to just load config "globally" once, and keep it as global. It avoid passing it to so many functions
I've done something like that:

```
import { Config, defaultConfig } from '../model/config';
import { getConfig } from '../api/routes';
import { getHTTPErrorDetails } from './errors';

export let config = defaultConfig;

export const loadConfig = async () => {
  try {
    config = await getConfig();
  } catch (err) {
    console.log(getHTTPErrorDetails(err));
  }
};

// Only for testing
export const setConfig = (cfg: Config) => {
  config = cfg;
};
```

I'll send a PR on top of yours next week, after some cleanup .. would be great if you can test it, making sure I'm not missing something